config-example.properties

```diff
@@ -42,6 +42,9 @@ graph.dataaccess=RAM_STORE
 # graph but less for every request.
 # prepare.chWeighting=fastest
 #
+# You can also setup multiple weightings, by providing a coma separated list like:
+# prepare.chWeighting=fastest, shortest
```

Comment:
I thought about that as well. What is the downside of trimming? We do not allow weighting names like `" name"` or `"name "`. The benefit is that our config files are not sensible to whitespace.

I would not expect a config file to be sensible to whitespaces.

> trimming inside the setter

Do you think adding logic to a setter is bad?